alexa-cookie.js

```diff
@@ -554,7 +554,7 @@ function AlexaCookie() {
                 request(options, (error, response, body) => {
                     if (!error) {
                         try {
-                            if (typeof body !== 'object') body = JSON.parse(body);
+                            if (body != '') body = JSON.parse(body);
```

Comment:
Ok this is more of a bandaid than a fix :)

The actual problem is the 401 response on `/api/users/me?platform=ios&version=${apiCallVersion}`
Apparently the cookie is wrong. If I do the call on properly extracted cookie I get all sorts of data about my account. (I have to do this on the alexa.amazon.DE domain for a German account!)

I didn't quite follow, why the call is needed - I am good without it 🙂